sr_ethercat_hand_config/controls/motors/lh/motor_board_effort_controllers.yaml

```diff
@@ -1,340 +1,380 @@
 ffj0:
   backlash_compensation: true
   pid:
-    d: 64
+    d: 0
     deadband: 5
     f: 300
     i: 0
-    imax: 550
+    imax: 0
     max_pwm: 600
     p: 6200
     sg_left: 0
     sg_right: 0
     sgleftref: 0
     sgrightref: 0
     sign: 0
+    torque_limit: 500
+    torque_limiter_gain: 64
   pos_filter:
     tau: 0.05
 ffj3:
   backlash_compensation: true
   pid:
-    d: 64
+    d: 0
     deadband: 5
     f: 300
     i: 0
-    imax: 450
+    imax: 0
     max_pwm: 600
     p: 6200
     sg_left: 0
     sg_right: 0
     sgleftref: 0
     sgrightref: 0
     sign: 0
+    torque_limit: 450
+    torque_limiter_gain: 64
   pos_filter:
     tau: 0.05
 ffj4:
   backlash_compensation: true
   pid:
-    d: 64
+    d: 0
     deadband: 5
     f: 300
     i: 0
-    imax: 350
-    max_pwm: 300
+    imax: 0
+    max_pwm: 400
     p: 6200
     sg_left: 0
     sg_right: 0
     sgleftref: 0
     sgrightref: 0
     sign: 0
+    torque_limit: 350
+    torque_limiter_gain: 64
   pos_filter:
     tau: 0.05
 lfj0:
   backlash_compensation: true
   pid:
-    d: 64
+    d: 0
     deadband: 5
     f: 300
     i: 0
-    imax: 550
+    imax: 0
     max_pwm: 600
     p: 6200
     sg_left: 0
     sg_right: 0
     sgleftref: 0
     sgrightref: 0
     sign: 0
+    torque_limit: 500
```

Comment:
Hello,
These values are estimations that we've been using historically and we didn't have problems with, I provided them to Dan. The only thing I would like to change is the TH1  & TH2 to 400, I 've just remember that OpenAI had 450 and the T-bar broke, so let's going to reduce TH1 and TH2 torque_limit to 400. The rest of the values should be OK.
Cheers,